hack/local-up-cluster.sh

```diff
@@ -742,6 +742,7 @@ EOF
     sudo "${GO_OUT}/hyperkube" proxy \
       --config=/tmp/kube-proxy.yaml \
       --master="https://${API_HOST}:${API_SECURE_PORT}" >"${PROXY_LOG}" \
+      --cluster-cidr=${SERVICE_CLUSTER_IP_RANGE} \
```

Comment:
I don't think this is actually what you're intending it to be - the `--cluster-cidr` argument to the kube-proxy should be the IP range used for Pod IPs, not the service cluster IP range.

Reply:
Also, I think we should be using the `/tmp/kube-proxy.yaml` configuration file for any new kube-proxy configuration parameters, rather than passing command line flags.